flexget/plugins/output/subtitles_subliminal.py

```diff
@@ -164,7 +167,7 @@ def on_task_output(self, task, config):
             # save subtitles to disk
             for video, subtitle in downloaded_subtitles.items():
                 if subtitle:
-                    subliminal.save_subtitles(video, subtitle, single=single_mode)
+                    subliminal.save_subtitles(video, subtitle, single=single_mode, directory=config.get('directory', None))
```

Comment:
might want to do os.path.expanduser(config.get('directory', ''). Just in case they decided to do '~/user/movies/subs'. subliminal requires absolute paths.